librz/il/theory_bitv.c

```diff
@@ -230,7 +230,7 @@ void *rz_il_handler_div(RzILVM *vm, RzILOpBitVector *op, RzILTypePure *type) {
 		if (rz_bv_is_zero_vector(y)) {
 			result = rz_bv_new(y->len);
 			rz_bv_set_all(result, true);
-			rz_il_vm_event_add(vm, rz_il_event_exception_new("division by zero"));
+			rz_il_vm_event_add(vm, rz_il_event_exception_new(RZ_IL_EVENT_EXC_DIV_ZERO, "division by zero"));
```

Comment:
i'm not sure if this makes a lot of sense. for me `rz_il_event_exception_new` was supposed to hold any human readable exception, and not something linked to a flag.